.github/workflows/interfacedata-updater.yml

```
@@ -39,6 +39,7 @@ jobs:
         token: ${{ secrets.GITHUB_TOKEN }}
         title: Update InterfaceData based on WebRef
         commit-message: Update InterfaceData based on WebRef
+        committer: mdn-content-bot <noreply@developer.mozilla.org>
```

The committer is already `GitHub <noreply@github.com>`, but the `author` isn't. So let's set the author to the same value as the committer:

```suggestion
        author: GitHub <noreply@github.com>
```

See:

```
commit 851c97e9efce9636f466b8aa9d2bbc04c4ea21bc (HEAD -> interfacedata-update, upstream/interfacedata-update)
Author: Josh-Cena <Josh-Cena@users.noreply.github.com>
Commit: GitHub <noreply@github.com>

    Update InterfaceData based on WebRef
```